### ISSUE TITLE
Hardware IDs for FTDI devices are not parsed correctly on Windows

### ISSUE THREAD

**Issue by shuxu94** [pyserial/pyserial#61]
When using serial.tools.list_ports.comports() to retrieve a list of ListPortInfo objects on windows. Those objects does not have the correct vid/pid/serial_number attributes. 
For reference the FTDI serial number should be A600ARJH and should be VID:PID 0403:6001
```
PS C:\Users\sxu> python -m serial.tools.list_ports -v
COM1
    desc: Communications Port (COM1)
    hwid: ACPI\PNP0501\1
COM6
    desc: USB Serial Port (COM6)
    hwid: FTDIBUS\VID_0403+PID_6001+A600ARJHA\0000
COM7
    desc: KitProg USB-UART (COM7)
    hwid: USB VID:PID=04B4:F139 LOCATION=1-1
3 ports found
```
comports returns:

```
Python 2.7.9 (default, Dec 10 2014, 12:28:03) [MSC v.1500 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> from serial.tools.list_ports import comports
>>> ports = comports()
>>> for port in ports:
...     print port.hwid
...
ACPI\PNP0501\1
FTDIBUS\VID_0403+PID_6001+A600ARJHA\0000
USB VID:PID=04B4:F139 LOCATION=1-1
>>> ports = comports()
>>> for port in ports:
...     print port.serial_number
...
None
None
None
>>>
```

**Comment by jrast** [comment]
When you talk about the wrong VID/PID/Serial, do you mean this line: 
``FTDIBUS\VID_0403+PID_6001+A600ARJHA\0000``? 

If yes, I think the problem is not related to pyserial as this string is directly taken from the ``SetupDiGetDeviceInstanceId`` or ``SetupDiGetDeviceRegistryProperty`` function from the ``setupapi`` library.

**Comment by zsquareplusc** [comment]
"normal" devices start with "USB" and `list_ports` detects that, then deconstructs that hardware ID into the fields such as shown by the other example (COM7). So it looks like there needs to be a special case because some driver writer decided to do things differently :/

**Comment by zsquareplusc** [comment]
Partial fix included in new release (v3.0.1), USB information is now added in the format used by other devices too. However the location information provided by Windows is hidden by the FTDI driver, so there is no `LOCATION` part (and I have currently no plan to spend time on this).

I do not know if there are additional chars in the serial number such as extra `A`s. With `hwgrep://` you can also match the partial serial number or just match what `list_ports` returns anyway.
